lib/whitehall_eu_exit_report.rb

```diff
@@ -0,0 +1,202 @@
+require "fileutils"
+require "ostruct"
+
+class WhitehallEuExitReport
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def initialize(path:)
+    @path = path
+  end
+
+  def call
+    organisations.find_each do |organisation|
+      puts organisation.title, organisation.content_id
+
+      slug = organisation.base_path[26..-1] # /government/organisations/
```

Comment:
Are we confident these don't contain any nasty characters?

Reply:
I've ran it locally with a full replication without any weird characters!